.github/workflows/build.yml

```
@@ -16,6 +16,8 @@ jobs:
       fail-fast: false
       matrix:
         ruby:
+          - 3.3.6
+          - 3.1.6
```

I know that we were already doing it this way, but I would also avoid specifying the patch version:

```suggestion
          - 3.3
          - 3.1
```

That way we ensure that we always tun the CI against the latest patch versions. Then, as a followup we can update the rest of the entries of the ruby matrix

---

Makes sense!

---

Done in [1b405fd](https://github.com/cedarcode/cose-ruby/pull/66/commits/1b405fd74c32edf9f4ce4c5f70d73d9934122276)